### ISSUE TITLE
Move to latest standard Android gitignore

### ISSUE THREAD

**Issue by ajburley** [hawknetwork/sealnote-plus#19]
Currently the gitignore is non-standard, even for a file-based Android Studio project. We should move to the standard Android gitignore, with changes only when absolutely necessary, and demarcated as such with comments.

**Comment by ajburley** [comment]
Here is the latest standard gitignore from Android Studio 3.4.2:

```
*.iml
.gradle
/local.properties
/.idea/caches
/.idea/libraries
/.idea/modules.xml
/.idea/workspace.xml
/.idea/navEditor.xml
/.idea/assetWizardSettings.xml
.DS_Store
/build
/captures
.externalNativeBuild
```

It's a bit more terse than I had expected!

**Comment by ajburley** [comment]
This page was really helpful in checking the gitignore was correct: https://proandroiddev.com/deep-dive-into-idea-folder-in-android-studio-53f867cf7b70

In the end, I did accept the Android Studio defaults, my only concern was gradle.xml but in the end I decided to accept the default behaviour (as while it's a pain for developers to all have to share the same gradle relative path, there may be other shared settings in the file).

The only changes I made from the default were to add individual module build directories (as the original excluded build directory at the root but not at module-level). I'm confident this is correct, as generated build directories should never be in version control.
